_data/registry.yml

```diff
@@ -200,7 +200,7 @@
       description: Allows clients to correlate requests with server responses
     - name: msgid
       specs: 
-        - msgid
+        - message-ids
```

Comment:
This particular change breaks the registry table (the reg table depends on `specs.yml` which uses the `msgid` spec name). Change this back to `msgid` and it'll build all fine.